openfasoc/generators/cryo-gen/tools/parse_rpt.py

```diff
@@ -1,16 +1,19 @@
 import re
 import subprocess
-import sys
+import sys, os
 
-drc_filename = "flow/reports/sky130hd/cyro/6_final_drc.rpt"
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+from common.check_gen_files import check_gen_files
+
+drc_filename = "flow/reports/sky130hd/cryo/6_final_drc.rpt"
```

Comment:
cryo-gen supports three libraries of sky130 - sky130hd, sky130hs, sky130hvl. Can you please update so that all three variants are checked?